fuel-relayer/src/validator_set.rs

```diff
@@ -0,0 +1,69 @@
+use std::collections::HashMap;
+
+use fuel_core_interfaces::relayer::RelayerDb;
+use fuel_tx::Address;
+
+pub struct CurrentValidatorSet {
```

Comment:
Are there any tests for this?

Reply:
This is covering old logic it is going to be fully rewritten.